scripts/config/tcl.py

```diff
@@ -254,10 +254,15 @@ def process_string(value: str, state: State) -> str:
             full_abspath = os.path.abspath(value)
 
             # Resolve globs for paths that are inside the exposed directory
-            if value.startswith("/") and full_abspath.startswith(found):
-                files = glob.glob(full_abspath)
-                files_escaped = [file.replace("$", r"\$") for file in files]
-                value = " ".join(files_escaped)
+            if value.startswith("/"):
+                if full_abspath.startswith(found):
+                    files = glob.glob(full_abspath)
+                    if not files:
+                        raise InvalidConfig(f"No such file {full_abspath}")
```

Comment:
Why not just a warning? The glob thing is best-effort. It shouldn't stop the whole flow. Maybe the user wants the path for something else down the line, i.e., it's going to be used by a later step. Let's not be presumptuous.